crates/arroyo-connectors/src/rabbitmq_stream/mod.rs

```diff
@@ -0,0 +1,276 @@
+use anyhow::{anyhow, bail};
+use arroyo_operator::connector::{Connection, Connector};
+use arroyo_operator::operator::OperatorNode;
+use arroyo_rpc::{api_types::connections::TestSourceMessage, OperatorConfig};
+use rabbitmq_stream_client::types::OffsetSpecification;
+use rabbitmq_stream_client::{Environment, TlsConfiguration};
+use serde::{Deserialize, Serialize};
+use typify::import_types;
+
+use crate::rabbitmq_stream::source::RabbitmqStreamSourceFunc;
+use crate::{pull_opt, ConnectionType};
+
+mod source;
+
+pub struct RabbitmqStreamConnector {}
+
+const CONFIG_SCHEMA: &str = include_str!("./profile.json");
+const TABLE_SCHEMA: &str = include_str!("./table.json");
+const ICON: &str = include_str!("./rabbitmq_stream.svg");
+
+import_types!(schema = "src/rabbitmq_stream/profile.json");
+import_types!(schema = "src/rabbitmq_stream/table.json");
+
+impl Connector for RabbitmqStreamConnector {
+    type ProfileT = RabbitmqStreamConfig;
+    type TableT = RabbitmqStreamTable;
+
+    fn name(&self) -> &'static str {
+        "rabbitmq_stream"
```

Comment:
I think just `rabbitmq` would be a better name, as it aligns with our other connectors that identify the system rather than the particular way they interact. So for example, we have a single `redis` connector that can read from various kinds of redis datastructures.

Reply:
I will also rename the `RabbitmqStreamConnector` struct to `RabbitmqConnector`. Ok?